defold-spine/editor/src/spineext.clj

```diff
@@ -382,6 +382,7 @@
                                                           (:texture render-args)))]
     (when-not is-selection-pass?
       (gl/set-blend-mode gl blend-mode))
+    (shader/set-uniform shader gl "world_view_proj" (:world-view-proj render-args))
```

Comment:
The fix (the rest just improved example to be able to reproduce the issue in our examples)